layouts/shortcodes/resources_common_properties.md

```diff
@@ -2,10 +2,11 @@ The following properties are common to every resource:
 
 `ignore_failure`
 
-:   **Ruby Type:** true, false \| **Default Value:** `false`
+:   **Ruby Type:** true, false, :quiet \| **Default Value:** `false`
 
     Continue running a recipe if a resource fails for any reason.
-
+    Using `:quiet` logs the full error though will still continue running the recipe.
```

Comment:
```suggestion

    Using `:quiet` logs the full error and the recipe will continue to run if a resource fails.
```

Reply:
I'm not sure if that's the correct description. Using :quiet prevents a full error stacktrace from being shown.

Reply:
So maybe

`:quiet` will not display the full stack trace and the recipe will continue to run if a resource fails.